pegen/testutil.py

```diff
@@ -76,6 +77,9 @@ def generate_parser_c_extension(
     Returns a module object with a parse_string() method.
     TODO: express that using a Protocol.
     """
+    # Check that the working directory is empty: reusing non-empty temporary
+    # directories when generating extensions can lead to segmentation faults.
```

Comment:
Link to the relevant issue in the comment. (I'm usually not a big fan of comments saying "fix issue #NNNNN" but in this case it seems impossible to summarize the issue concisely.